.gitignore

```diff
@@ -9,3 +9,8 @@ __pycache__
 ideafast-*-full.csv
 *.json
 ucam_db.csv
+
+# mypy
+.mypy_cache/
+.dmypy.json
+dmypy.json
```

Comment:
Can you ignore `.DS_Store` files and from the 3 from this PR (`**/.DS_Store`)? They're autogenerated by OSX

Reply:
Woops - an oversight on my end